scripts/core/EXT_Template.rst

```diff
@@ -0,0 +1,42 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+:orphan:
+
+==================================
+Unified Runtime Extension Template
+==================================
+
+API
+---
+
+* Enumerations
+
+
+    * List all enumerations defined by the extension.
+
+
+* Structures
+
+
+    * List all structures defined by the extension.
+    * ...
+
+
+* Functions
+
+
+    * List all functions defined by the extension.
+    * ...
+
+
+Unified Runtime Extension Template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this section describe in detail the purpose of the extension, along with its
+valid usage.
```

Comment:
Quite like it when extensions have a version history at the bottom, could we add that to the template?

Reply:
This is useful info, I do wonder if having the ability to view different extension versions side-by-side would also be useful though?

Reply:
That would be really good. @kbenzie I've seen versioned docs before on the Zephyr project [link](https://docs.zephyrproject.org/latest/) - the is a dropdown in the bottom left where you can select the version of the API you want to view. I'm not really sure how this is implemented, but perhaps we can have something like that as well as a changelog (like ewan suggested).

Reply:
Looking at the Zephyr page I think that's for the entire page rather than just the extension specs, my gut feeling is you'd need to have a separate Sphinx build for each extension to get something exapctly like that.